CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- Change `Version()` function in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp` to a `const Version` string. (#8142)
```

Comment:
@open-telemetry/go-approvers I think we could make it more visible this is a breaking change. But at the same time, doing something too prominent for the version string probably too much. WDYT?

Reply:
I prefer to only mark content or tags related to significant changes here.

Reply:
You mean do we even need to have the `Version` const provided? Every other instrumentation has it (as a function actually, not a string).